example_run_vehicle_attributes_recognition_barrier.py

```diff
@@ -0,0 +1,142 @@
+import sys
```

Comment:
This runs for a bit, but classifies me as a red car when I'm wearing a blue sweatshirt.

After a minute or so, it crashes with:

```
ValueError: array is too big; `arr.size * arr.dtype.itemsize` is larger than the maximum possible size.
Traceback (most recent call last):
  File "example_run_vehicle_attributes_recognition_barrier.py", line 121, in <module>
    data0 = data[0,:,:]
IndexError: too many indices for array
Stopping threads: ...
```

Reply:
This one was done by yurii.
I just merged his latest work with `multiple_outpout_tensors` branch.

Reply:
Inference still seems incorrect:

<img width="297" alt="image" src="https://user-images.githubusercontent.com/7880/75814828-a9281f80-5d4f-11ea-8ccb-02cc68cd1b8b.png">

...I am pale/white for sure, but not a car :).

Good news is it hasn't crashed for me over a period of a couple of minutes ... going to let it run longer.

Reply:
I’d be curious if we fed that same image into the same network running on a host computer for inference it it would have the same output.